eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -226,4 +226,8 @@
 
   <!-- Suppress external dependency check for jacksonwrapper overriding deserialize method -->
   <suppress checks="com.azure.tools.checkstyle.checks.ExternalDependencyExposedCheck" files="com.azure.core.implementation.serializer.jsonwrapper.jacksonwrapper.JacksonDeserializer.java"/>
+
+  <!-- Suppress ConfigurationCredentialsPolicy for HttpPipelinePolicyCheck.
+   This suppression need to be deleted after AzConfig using AAD credential. -->
+  <suppress checks="com.azure.tools.checkstyle.checks.HttpPipelinePolicyCheck" files="com.azure.data.appconfiguration.implementation.ConfigurationCredentialsPolicy.java"/>
```

Comment:
Is there an issue associated with adding AAD support? I can see this being lost.

Reply:
https://github.com/Azure/azure-sdk-for-java/issues/5131

Reply:
Which is not supported in current release

Reply:
Yes. Can you link to this in the xml?